core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -529,11 +532,17 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
                     <div class="col-sm-11">
                       <div>
                         <span><strong><[message.authorUsername]></strong></span>
+                        <span style="float: right; margin-right: 15px;"
+                              ng-if="message.messageId !== 0 && message.updatedStatus">
+                          <em translate="I18N_LEARNER_DASHBOARD_FEEDBACK_THREAD_STATUS_CHANGE_MESSAGE"
+                              translate-values="{threadStatus: getHumanReadableStatus(message.updatedStatus)}"></em>
+                        </span>
+                        <div ng-if="message.messageId !== 0 && message.updatedStatus">
```

Comment:
I'm not yet done with this PR. Could you please wait till I send a message telling it's ready for another review?